package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-loader",
-  "version": "0.16.1",
+  "version": "0.16.2",
```

Comment:
Please don't 😛, we use `standard-version` for release management across webpack-contrib and this is the responsibility of the respective maintainer